webpack/assets/javascripts/react_app/components/PermissionDenied/PermissionDenied.test.js

```diff
@@ -0,0 +1,25 @@
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import React from 'react';
+import PermissionDenied from '.';
+
+describe('PermissionDenied', () => {
```

Comment:
Can you write your test using the `testComponentSnapshotsWithFixtures`?
See:
https://sharvit.github.io/react-redux-test-utils/manual/unit-testing-components.html
Example: https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/Template/__test__/ReportTemplateInput.test.js